definitions/scenarios/upgrade_to_satellite_6_3.rb

```diff
@@ -0,0 +1,77 @@
+module Scenarios::Satellite_6_3
+  class Abstract < ForemanMaintain::Scenario
+    def self.upgrade_metadata(&block)
+      metadata do
+        tags :upgrade_to_satellite_6_3
+        confine do
+          feature(:downstream) && feature(:downstream).current_minor_version == '6.2'
+        end
+        instance_eval(&block)
+      end
+    end
+  end
+
+  class PreUpgradeCheck < Abstract
+    upgrade_metadata do
+      description 'checks before upgrading to Satellite 6.3'
+      tags :pre_upgrade_checks
+      run_strategy :fail_slow
+    end
+
+    def compose
+      add_steps(find_checks(:default))
+      add_steps(find_checks(:pre_upgrade))
+    end
+  end
+
+  class PreMigrations < Abstract
+    upgrade_metadata do
+      description 'procedures before migrating to Satellite 6.3'
+      tags :pre_migrations
+    end
+
+    def compose
+      add_steps(find_procedures(:pre_migrations))
+    end
+  end
+
+  class Migrations < Abstract
+    upgrade_metadata do
+      description 'migration scripts to Satellite 6.3'
+      tags :migrations
+    end
+
+    def compose
+      add_step(Procedures::Repositories::Setup.new(:version => '6.3'))
+      add_step(Procedures::Packages::Update.new(:assumeyes => false))
+      add_step(Procedures::Installer::Upgrade.new)
+      add_steps(find_procedures(:label => :installer_upgrade))
```

Comment:
Please note that this is actually a duplicate line (my mistake). I'm fixing it in the original code here https://github.com/theforeman/foreman_maintain/pull/88/files#diff-b8fa89d315b87599a0ce9bd5f2bffda3L48 but let's not copy the mistake over other places